controlplane/src/bin/delete-user.ts

```diff
@@ -80,11 +90,14 @@ if (!isSafe) {
 }
 
 // Delete the user
-await userRepo.deleteUser({
-  id: user.id,
-  keycloakClient,
-  keycloakRealm: realm,
-});
+await userRepo.deleteUser(
+  {
+    id: user.id,
+    keycloakClient,
+    keycloakRealm: realm,
+  },
+  blobStorage,
```

Comment:
Any reason why is this service dependency not part of he UserRepository constructor?

Reply:
This would require refactoring way too many files since we have to touch other repos as well. Let's not block the PR